atat/app.py

```diff
@@ -88,6 +89,11 @@ def make_app(config):
     if ENV != "prod":
         app.register_blueprint(dev_routes)
 
+    if (ENV == "dev" or ENV == "development") and app.config[
+        "DEV_DEBUG_TOOL"
+    ] == "True":
+        debug_tools_bar(app)
+
```

Comment:
Let's break out these names to make them slightly less "magic". Or use a regex and accept any environment that starts with `dev`.

```suggestion
    dev_env_names = ("dev", "development")
    if (ENV in dev_env_names) and app.config["DEV_DEBUG_TOOL"] == "True":
        debug_tools_bar(app)

```

Reply:
The standard name for the environment for development by the documentation of Flask should be `development.` 

But for some reason, everywhere in the code, the old team uses `dev.` 

I am trying to have a working theory of why they name this in a custom matter. At this moment, I don't know why. But if you can give me some inside about that, maybe in the ticket. 

> base on https://flask.palletsprojects.com/en/1.1.x/cli/#watch-extra-files-with-the-reloader

So, for me would be better to only have it on `development` (that exact word only), but if you read in the code there are some service been provided for the `dev` (also an exact word), so I try to make available for both.  

I am also open to plan this better with you later, probably a 15 min meeting.

Reply:
Yep! I see that the goal is to make it work for both `dev` and `development` as the code is written; this small refactoring would accomplish the same thing but actually assign a "name" to those strings.

I feel we should have a discussion as part of a separate ticket whether we can use `development` instead of `dev`.